src/app/tests/suites/TestFabricSyncBridgedNode.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Test Fabric Synchronization condition on Bridged Node Device Type
+
+PICS:
+    - MCORE.BRIDGE.FS
```

Comment:
nit I think this is supposed to be `MCORE.FS`

Reply:
Hi, as far as I know this has been discussed with Chris Letnick and as per him `MCORE.BRIDGE.FS` makes sense so we can retain the same since we are verifying the FS feature specifically on a Bridge node.

Reply:
The bridge and admin are one DUT. You can't test one without the other. I would argue we should simplify the PICS and just use MCORE.FS. That is my 2cents

Reply:
@tehampson agree It's just that we wanted to be more specific so that the PICS name itself clarifies that we are verifying FS on bridge node in particular. @cletnick thoughts ?